lamindb_setup/core/_settings_storage.py

```diff
@@ -59,7 +59,7 @@ def mark_storage_root(root: UPathStr):
     # we need to touch a 0-byte object in the storage location to avoid
     # permission errors from leveraging s3fs on an empty subdirectory
     root = convert_pathlike(root)
-    root.fs.touch(str(root / IS_INITIALIZED_KEY))  # type: ignore
+    root.touch(str(root / IS_INITIALIZED_KEY))  # type: ignore
```

Comment:
String casting should no longer be needed, now, meaning, you should be able to remove `str()`